packages/keymaster/src/db-wallet-json.js

```diff
@@ -17,10 +17,16 @@ export function saveWallet(wallet, overwrite = false) {
 }
 
 export function loadWallet() {
-    if (fs.existsSync(walletName)) {
-        const walletJson = fs.readFileSync(walletName);
-        return JSON.parse(walletJson);
+    if (!fs.existsSync(walletName)) {
```

Comment:
Should this file be reverted to its original state?

Reply:
It now handles the error of loading an encrypted wallet.

Reply:
OK, good